tensorflow/compiler/xla/service/gpu/kernel_mapping_scheme.h

```diff
@@ -76,19 +76,33 @@ namespace gpu {
 class KernelMappingScheme {
  public:
   enum { DimZ = 0, DimY, DimX, DimTot };
+  enum IndexingOrder {
+  // Threads read consecutive elements.
+    LinearIndexingX,
+  // Threads read strided elements while keeping memory coalescing.
+    StridedIndexingX,
+  // Threads read a few consecutive elements then take a strided
+  // step. This can trigger vectorized reads and keep memory
+  // coalescing.
+    LinearStridedIndexingX
```

Comment:
Maybe `UnrolledStridedIndexingX`?

Reply:
Personnally, I prefer the name I pick. But if you prefer your, I can change it.
I do not like using Unrolled as the code is unrolled for the 3 case. So we should put it in all names or no names.
As this is a mix  of Linear and Strided indexing, I like putting the 2 names to make it clear.

But I could change the order of the name to StridedLinearIndexingX. The Linear part is inside, so the part name order would match the loop order.

It could also be LinearVectorizedIndexingX, but as the vectorization isn't done by XLA, I still perfer StridedLinearIndexingX.

Other idea? What do you think of the new name suggestion?